tests/bandagetests.cpp

```diff
@@ -34,6 +34,13 @@ class BandageTests : public QObject
 {
     Q_OBJECT
 
+public:
+    BandageTests() {
+
+        std::cout << "sizeof(DeBruijnNode)=" << sizeof(DeBruijnNode)
```

Comment:
It's kinda debugging info. Maybe remove it or move it under `#ifdef`?

Reply:
It's part of tests. So, I'd keep it – we can see the stuff in the logs.